src/main/java/seedu/typed/model/ModelManager.java

```diff
@@ -149,8 +149,9 @@ public String toString() {
         @Override
         public boolean run(ReadOnlyTask task) {
             return nameKeyWords.stream()
-                    .filter(keyword -> StringUtil.containsWordIgnoreCase(task.getName().getValue(), keyword)).findAny()
-                    .isPresent();
+                    .filter(keyword -> StringUtil
+                    .containsFuzzyWordIgnoreCase(task.getName().getValue(), keyword))
```

Comment:
Yeah I noticed that as well, but left it in the same way AddressBook-Level4 originally implemented their find for the time being. I'll shift things around a bit and see what I can do to the restructuring.